src/main/java/net/snowflake/ingest/streaming/internal/ParquetRowBuffer.java

```diff
@@ -102,7 +102,11 @@ public void setupSchema(List<ColumnMetadata> columns) {
         this.statsMap.put(
             column.getInternalName(),
             new RowBufferStats(
-                column.getName(), column.getCollation(), column.getOrdinal(), null /* fieldId */));
+                column.getName(),
+                column.getCollation(),
+                column.getOrdinal(),
+                null /* fieldId */,
+                clientBufferParameters.getIsIcebergMode()));
```

Comment:
we are in !getIsICebergMode (see line 100 above), so can just hardcode false here.

Reply:
also applies to line 124.

Reply:
Switch to `getEnableDistinctValues()`